shared/run_diff.go

```diff
@@ -24,7 +24,7 @@ import (
 type DiffAPI interface {
 	GetRunsDiff(before, after TestRun, filter DiffFilterParam, paths mapset.Set) (RunDiff, error)
 	GetDiffURL(before, after TestRun, diffFilter *DiffFilterParam) *url.URL
-	GetMasterDiffURL(sha string, product ProductSpec) *url.URL
+	GetMasterDiffURL(sha string, product ProductSpec, diffFilter *DiffFilterParam) *url.URL
```

Comment:
Why the change of function signature? Is there (or will there be) any call of `GetMasterDiffURL` that specifies non-nil diff filter param?

Reply:
Consistency.